### ISSUE TITLE
Is it possible de modify registration delay ?

### ISSUE THREAD

**Issue by ludivinefafournoux** [VoIPGRID/VialerSIPLib#211]
### Expected behavior

I want to modify the delay between two registrations

### Actual behavior

Now the delay is 300 seconds, after the first retry

**Comment by mudassirzulfiqar** [comment]
I managed to implement this yesterday.
Go to VSLAccount.m file 
Find `configureWithAccountConfiguration` method
add these below lines some where in this method
```
  acc_cfg.reg_timeout = 9000;
  acc_cfg.reg_retry_interval = 5;
  acc_cfg.reg_retry_random_interval = 5;
```

**Comment by ludivinefafournoux** [comment]
yes I did it, but i'm trying to add it in my SipUser like sipRegisterOnAdd, but no success now 

I've added property in `VSLAccountConfiguration.h`:
`@property (nonatomic) unsigned int retryIntervalSec;`

In `VialerSIPLib.h`:
`@property (readonly, nonatomic) unsigned int retryIntervalSec;`

And in my `SipUser` init I've added this parameter, and I put the value in Keys but I have the value 0 ! I don't understand (It's the first time I'm doing Objective-C)

**Comment by mudassirzulfiqar** [comment]
Why did you include in `VSLAccountConfiguration.h` ?
I used these members in my `VSLAccount.m` and Im done. This is now Global Settings. Every register account will be have same Retry Intervals.

**Comment by ludivinefafournoux** [comment]
Yes, but I would like this to be configurable from the SipUser, like other parameters and maybe add it to the lib
